docs/source/dev_guide/releasing-e3sm-diags.rst

```diff
@@ -161,120 +161,29 @@ This will have the ``conda/meta.yaml`` we edited in the first section.
 Releasing The Software On Anaconda
 ----------------------------------
 
-1. Since we're building with ``noarch``, you can run the below steps on
-either a Linux or macOS machine. You **do not** need to run this steps on both.
+1. The CI/CD workflow takes care of this step. Be sure to tag a release.
+See :ref:`Continuous Integration / Continuous Delivery (CI/CD) <ci-cd>`.
```

Comment:
@tomvothecoder How do you tag a release?

Reply:
On GitHub's site, you go here and tag the release: https://github.com/E3SM-Project/e3sm_diags/releases

Reply:
On GitHub's site, you go here and make a release, which creates the tag: https://github.com/E3SM-Project/e3sm_diags/releases

Reply:
Oh than that's just the https://e3sm-project.github.io/e3sm_diags/_build/html/master/dev_guide/releasing-e3sm-diags.html#creating-a-release-on-github task. Ok I'll change the tag to that section.